src/sidebar/util/time.js

```diff
@@ -209,14 +209,13 @@ function decayingInterval(date, callback) {
  * Formats a date as a string relative to the current date.
  *
  * @param {number} date - The absolute timestamp to format.
+ * @param {Date} [now] - The date to consider as the current time.
  * @return {string} A 'fuzzy' string describing the relative age of the date.
  */
-function toFuzzyString(date, Intl) {
+function toFuzzyString(date, now = new Date(), Intl) {
```

Comment:
It's weird to me that this works but I guess that's a difference between js and python. In python this wouldn't create a new Date every time but looks like that's not the case in js.

Reply:
What is `Intl` here-shouldn't that be documented as a param?

Reply:
`Intl` here is the browser's `Intl` global object which is a namespace for internationalization objects. Yes it should be documented :P

Reply:
> It's weird to me that this works but I guess that's a difference between js and python.

Yes indeed. In JS the default value is evaluated every time. I think this is much more useful than Python 😉